templates/hedy-page/editor-and-output.html

```diff
@@ -15,7 +15,8 @@
       <!-- The actual editor -->
       <div id="editor" data-cy="editor" style="background: #272822; font-size:0.95em; color: white; direction: {{ g.dir }};" lang="en" blurred='false'
         {% if editor_readonly %}data-readonly="true"{% endif %}
-        class="w-full flex-1 text-lg rounded min-h-0 overflow-hidden fixed-editor-height"></div>
+        class="w-full flex-1 text-lg rounded min-h-0 overflow-hidden fixed-editor-height"
+           data-kwlang="{{ current_language().lang }}" data-level="{{ level }}"></div>
```

Comment:
An enclosing scope should be good enough, and I think it should be on there already, no?